charts/llm-d-modelservice/templates/epp-deployment.yaml

```diff
@@ -61,8 +63,6 @@ spec:
           periodSeconds: 10
           successThreshold: 1
           failureThreshold: 3
-        {{- end }}
-        {{- if (not .Values.routing.epp.disableLivenessProbe) }}
```

Comment:
These should be dissable-able if users want, please add that back

Reply:
My bad, editor misguided >.<